examples/text-generation/README.md

```diff
@@ -218,36 +218,67 @@ python run_generation.py \
 Another way to simulate dynamic input is to use `--simulate_dyn_prompt`. For example `--simulate_dyn_prompt 25,35,45` will extend or crop the default prompt (or the prompt passed in using `--prompt`) to sizes 25, 35, and 45, and throughput will be measured for these 3 lengths. If `--simulate_dyn_prompt` is used, the min and max input lengths from it are computed to perform warmup as well. One final optimization that can be used in case of dynamic inputs is `--reduce_recompile`. Thus the suggested configuration to simulate dynamicity after warmup is to use all three arguments: `--simulate_dyn_prompt 25 35 45 --reduce_recompile --bucket_size 30`
 ### Running with FP8
 
-Llama2-7b in FP8 is enabled. Use `--fp8` to enable quantization in fp8.
-Add the `--kv_cache_fp8` argument to run the model with a KV cache allocated in fp8.
+Llama2-70b and Llama2-7b in FP8 are enabled using the Quantization Toolkit (HQT), which provides model measurement and quantization capabilities in PyTorch.
+
 More information on enabling fp8 in SynapseAI is available here:
 https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html
```

Comment:
need to make sure the page is updated for 1.14 first. currently its not merged yet by the TR

Reply:
we are in the processing of update, it sould be done in a day or 2.

Reply:
Done